pkg/tsdb/intervalv2/intervalv2_test.go

```diff
@@ -69,6 +69,8 @@ func TestRoundInterval(t *testing.T) {
 		interval time.Duration
 		expected time.Duration
 	}{
+		{"10ms", time.Millisecond * 10, time.Millisecond * 1},
+		{"30ms", time.Millisecond * 30, time.Millisecond * 20},
```

Comment:
Seems like this is just accidentally duplicated?

Reply:
Sorry, I missed this. It's supposed to be the case for 15ms interval, rounding down to 10ms.